src/index.tsx

```diff
@@ -122,7 +124,7 @@ function CountryPhoneInput({
   return (
     <Input
       {...inputProps}
-      className="antd-country-phone-input"
+      className={`antd-country-phone-input ${className}`}
```

Comment:
Could you add a demo in `example` folder and describe why this change is necessary? I am glad to tell you how to write a demo doc if you're not familiar with `dumi`.

Reply:
I'm currently using this antd-country-phone-input with other UI framework(like Tailwind UI)
As you can see the style of the phone input element is slightly different so I need to add custom class.

![image](https://user-images.githubusercontent.com/19547819/131870380-ad1634a9-ff22-455b-ab2f-710b14fa152b.png)

will add a demo soon

Reply:
I see. It's necessary to work with CSS framework. How about this?
```
className={'antd-country-phone-input' + className ? ` ${className}` : ''}
```

Reply:
Looks good.
updated and added it to the basic-usage demo.
I'm not sure if I need to do other things? I haven't used `dumi` before.

Reply:
That's enough. Thanks for your work.